src/audio/module_adapter/module/dts.c

```diff
@@ -316,7 +316,7 @@ static int dts_codec_apply_config(struct processing_module *mod)
 		/* If param->size is less than param_header_size, then this param is not valid */
 		if (param->size < param_header_size) {
 			comp_err(dev, "dts_codec_apply_config() param is invalid");
-			return -EINVAL;
+			return 0;
```

Comment:
I don't remember about this. What is this for?

Reply:
DTS can't be loaded successfully without this change.

Reply:
ACK